packages/ripple-nuxt-tide/lib/core/componentLoader.js

```diff
@@ -1,5 +1,15 @@
 import coreComponents from './../config/tide.load-components'
-// import customComponentsLoadConfig from '~/tide.load-components.js'
+let customComponents = {}
+try {
+  const customComponentsLoadConfig = require('~/tide/tide.load-components.js').default
+  if (customComponentsLoadConfig) {
+    customComponents = {
+      ...customComponentsLoadConfig
+    }
+  }
+} catch (error) {
+  // no custom components
```

Comment:
@dylankelly have you tried this? Will it still throw errors in console? I remembered this catch won't prevent of that.

Reply:
Yep I tried this without the file and adding the try catch block works fine, I dont think we need to console anything here, we don't need to know if there aren't any custom components